client/my-sites/plan-features/index.jsx

```diff
@@ -459,20 +458,24 @@ PlanFeatures.defaultProps = {
 export default connect(
 	( state, ownProps ) => {
 		const { isInSignup, placeholder, plans, onUpgradeClick } = ownProps;
+		const selectedSiteId = isInSignup ? null : getSelectedSiteId( state );
+		const sitePlan = selectedSiteId && getSitePlan( state, selectedSiteId );
+		const hasSitePlanOwner = sitePlan && sitePlan.owner_id > 0;
```

Comment:
Method name should be `isCurrentUserSitePlanOwner( state, siteId );`, but other than that I agree. I thought about that, but I wasn't sure if it is enough logic to create a selector :)